daprdocs/content/en/js-sdk-docs/_index.md

```diff
@@ -0,0 +1,193 @@
+---
+type: docs
+title: "Dapr JavaScript SDK"
+linkTitle: "JavaScript"
+weight: 1000
+description: JavaScript SDK packages for developing Dapr applications
+---
+
+## Pre-requisites
+
+- [Dapr CLI]({{< ref install-dapr-cli.md >}}) installed
+- Initialized [Dapr environment]({{< ref install-dapr-selfhost.md >}})
+- [Node version 15 or greater](https://nodejs.org/en/) 
+
+## Installing and importing Dapr's JS SDK
+
+
+Install the SDK with npm:
+```
+npm i @roadwork/dapr-js-sdk
+```
+
+Import the libraries for the the given protocol you're using:  
+
+```javascript
+import { DaprClient, DaprServer } from "@roadwork/dapr-js-sdk/http";
+// OR (depending on the protocol)
+import { DaprClient, DaprServer } from "@roadwork/dapr-js-sdk/grpc";
+```
+
+## Building blocks
+
+The JavaScript SDK allows you to interface with all of the [Dapr building blocks]({{< ref building-blocks >}}).
+
+### Invoke a service
+
+```javascript
+import { DaprClient, HttpMethod } from "@roadwork/dapr-js-sdk/http";
+
+const daprHost = "127.0.0.1";
+const daprPort = "50050"; 
+
+async function start() {
+  const client = new DaprClient(daprHost, daprPort);
+
+  //POST Request
+  const response = await client.invoker.invoke(SERVICE_TO_INVOKE, METHOD_TO_INVOKE, HttpMethod.POST, {
```

Comment:
It's actually the APP ID of the application to invoke. What about writing this like the below?

```javascript
const serviceAppId = "my-dapr-app-id";
const serviceMethod = "say-hello";
const response = await client.invoker.invoke(serviceAppId , serviceMethod , HttpMethod.POST, {
```